ethcore/evm/src/interpreter/mod.rs

```diff
@@ -795,7 +795,7 @@ impl<Cost: CostType> Interpreter<Cost> {
 						TWO_POW_96 => a >> 96,
 						TWO_POW_224 => a >> 224,
 						TWO_POW_248 => a >> 248,
-						_ => a.overflowing_div(b).0,
+						_ => a/b,
```

Comment:
`a / b`?

Reply:
why is it changed?

Reply:
@dvdplm Please all those cases back. `Div` panics on default but we need overflowing behavior here, otherwise it breaks consensus.

Reply:
It seems like these methods were removed when going from `bigint` to `uint`, which makes sense since for unsigned integers div and rem should never overflow.

Reply:
I also had the same question though so it's better if @dvdplm confirms.

Reply:
Yeah right. I think it's this: https://github.com/paritytech/parity-common/pull/32

Reply:
Confirmed, that's the PR. I think the reason they were there to begin with was an attempt to fulfill a trait that is no longer in rust (removed ~1.13?).